index.html

```diff
@@ -16,6 +16,11 @@
       <!-- create button -->
       <div class="create">
         <h1>Add Tasks</h1>
+        <!-- <div name="todos" class="filter-todo">
+          <button value="all">All</button>
+          <button value="incompleted">Incomplete</button>
+          <button value="completed">Complete</button>
+        </div> -->
```

Comment:
We can always remove comments if they are unnecessary.

Reply:
done!